marklogic-data-hub-central/src/test/java/com/marklogic/hub/curation/controllers/FlowControllerTest.java

```diff
@@ -177,6 +178,39 @@ void runFlow() {
         JsonNode job = jobsController.getJob(resp.getJobId());
         String jobStatus = job.get("jobStatus").asText();
         assertEquals("finished", jobStatus, "Job status should be 'finished' once threads complete; job doc: " + job);
+
+
+        addStagingDoc("input/mapInput.json", "/input/mapInput.json", "default-ingestion");
+
+        //Equivalent to adding a step from GUI
+        controller.createStep("testFlow" , 2, "{\n" +
```

Comment:
If the purpose of this test is to run a flow and not to test the creation of a step, I think it's better just to add the step to the test flow file.